src/app/showcase/detail/[id]/panel/ShowcaseViewer.style.ts

```diff
@@ -0,0 +1,55 @@
+export const mobileViewer = {
+  width: '100vw',
+  boxSizing: 'border-box',
+  gridTemplateRows: 'auto',
+  gridTemplateColumns: 'auto',
+  gridTemplateAreas: `
+    "teamName"
+    "img"
+    "startEndDateViewer"
+    "skillInput"
+    "teamMembers"
+    "linksViewer"
+    "toastViewer"
+  `,
+  '@media screen and (min-width: 768px)': {
+    gridTemplateAreas: `
+      "img"
+      "teamName"
+      "startEndDateViewer"
+      "skillInput"
+      "teamMembers"
+      "linksViewer"
+      "toastViewer"
+    `,
+  },
+}
+
+export const pcViewer = {
+  display: 'flex',
+  flexDirection: 'column',
+  margin: '0 auto',
+}
+
+export const InformationViewerBox = {
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'row',
+  gap: '1rem',
+}
+
+export const imageViewer = {
+  maxWidth: '100%',
+  width: '100%',
+  height: '13.6rem',
+  gridArea: 'img',
+  borderRadius: '0.25rem',
+  border: '1px solid #2C2E35',
```

Comment:
스타일 객체를 SxProps로 타입을 지정하면 컬러팔레트에 있는 색상을 사용할 수 있습니다 참고해주세요.
```suggestion
  border: '1px solid',
  borderColor: 'line.alternative'
```